javascript-bank/bank.js

```diff
@@ -0,0 +1,40 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) && balance > 0) {
+    var newAccount = new Account(
+      this.accounts.number = this.nextAccountNumber,
+      this.holder = holder
+    );
```

Comment:
Like the comment above, `this` refers to the new `Bank` object here, not the `Account` object being created.